300-quarkus-vertx-webClient/src/main/java/io/quarkus/qe/vertx/webclient/ChuckNorrisResource.java

```diff
@@ -0,0 +1,89 @@
+package io.quarkus.qe.vertx.webclient;
+
+import io.quarkus.qe.vertx.webclient.config.ChuckEndpointValue;
+import io.quarkus.qe.vertx.webclient.config.VertxWebClientConfig;
+import io.smallrye.mutiny.Uni;
+import io.vertx.core.json.Json;
+import io.vertx.mutiny.core.Vertx;
+import io.vertx.mutiny.ext.web.client.WebClient;
+import io.vertx.mutiny.ext.web.client.predicate.ResponsePredicate;
+import io.vertx.mutiny.ext.web.codec.BodyCodec;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import javax.annotation.PostConstruct;
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.time.Duration;
+import java.util.Arrays;
+
+@Path("/chuck")
+public class ChuckNorrisResource {
+
+    @Inject
+    Vertx vertx;
+
+    @ConfigProperty(name = "chucknorris.api.domain")
+    private ChuckEndpointValue chuckNorrisQuote;
```

Comment:
2020-11-06 13:44:11,214 INFO  [io.qua.arc.pro.BeanProcessor] (build-2) Found unrecommended usage of private members (use package-private instead) in application beans:
	- @Inject field io.quarkus.qe.vertx.webclient.ChuckNorrisResource#chuckNorrisQuote